bootstrap/node.go

```diff
@@ -37,6 +37,8 @@ func NewNode(
 	benchmarkConsensusRoundRetryIntervalMillis uint32, // TODO: move all of the config from the ctor, it's a smell
 	transport gossipAdapter.Transport,
 	stateHistoryRetentionInBlockHeights uint64,
+	querySyncGraceBlockDist uint64,
```

Comment:
as discussed, these should probably be defined as a `Config` interface in the `BlockTracker`. Please open an issue for that and fix it ASAP.